auth/bucket_policy.go

```diff
@@ -47,19 +47,19 @@ func (bp *BucketPolicy) Validate(bucket string, iam IAMService) error {
 	return nil
 }
 
-func (bp *BucketPolicy) isAllowed(principal string, action Action, resource string) bool {
+func (bp *BucketPolicy) isAllowed(principal string, action Action, resource string) (isAllowed bool) {
```

Comment:
can we do with without the named/naked return? style-wise we have been avoiding naked returns.